sympy/integrals/tests/test_integrals.py

```diff
@@ -1344,37 +1343,34 @@ def test_issue_2884():
 
 
 def test_issue_8368():
-    assert integrate(exp(-s*x)*cosh(x), (x, 0, oo)) == \
-        Piecewise(
-            (   pi*Piecewise(
-                    (   -s/(pi*(-s**2 + 1)),
-                        Abs(s**2) < 1),
-                    (   1/(pi*s*(1 - 1/s**2)),
-                        Abs(s**(-2)) < 1),
-                    (   meijerg(
-                            ((S.Half,), (0, 0)),
-                            ((0, S.Half), (0,)),
-                            polar_lift(s)**2),
-                        True)
-                ),
-                And(
-                    Abs(periodic_argument(polar_lift(s)**2, oo)) < pi,
-                    cos(Abs(periodic_argument(polar_lift(s)**2, oo))/2)*sqrt(Abs(s**2)) - 1 > 0,
-                    Ne(s**2, 1))
-            ),
-            (
-                Integral(exp(-s*x)*cosh(x), (x, 0, oo)),
-                True))
-    assert integrate(exp(-s*x)*sinh(x), (x, 0, oo)) == \
-        Piecewise(
-            (   -1/(s + 1)/2 - 1/(-s + 1)/2,
-                And(
-                    Ne(1/s, 1),
-                    Abs(periodic_argument(s, oo)) < pi/2,
-                    Abs(periodic_argument(s, oo)) <= pi/2,
-                    cos(Abs(periodic_argument(s, oo)))*Abs(s) - 1 > 0)),
-            (   Integral(exp(-s*x)*sinh(x), (x, 0, oo)),
-                True))
+    got = integrate(exp(-s*x)*cosh(x), (x, 0, oo))
+    x0 = 1/pi
+    x1 = s**2
+    x2 = Abs(x1)
+    x3 = s > -oo
+    x4 = Abs(arg(s))
+    x5 = 2*x4 < pi
+    x6 = sqrt(x2)*cos(x4) - 1 > 0
+    cse_ans = [
+    Piecewise((pi*Piecewise((-s*x0/(1 - x1), x2 < 1), (x0/(s*(1 - 1/x1)),
+    1/x2 < 1), (meijerg(((1/2,), (0, 0)), ((0, 1/2), (0,)),
+    polar_lift(s)**2), True)), (x3 & x5 & x6 & (s > 1)) | (x3 & x5 & x6 &
+    (s < -1)) | (x3 & x5 & x6 & (s > -1) & (s < 1))),
+    (Integral(exp(-s*x)*cosh(x), (x, 0, oo)), True))]
+    assert got == cse_ans[0]
```

Comment:
This part of the diff makes me a little nervous. I can't tell how much the result has actually changed just from the diff here.

Reply:
I tested the old and new expressions with a few random() numbers.

Reply:
```python
for i in range(100):
  p = Rational(randint(1,30),randint(1,30))
  if not e(p):
   print(p)
   break
```

Reply:
But that second expression has a cosh instead of a sinh...checking

Reply:
doh - checked the first against the second. When checking each against the correct old Piecewise and the new cse, the loop never emits.